R/pkg/R/SQLContext.R

```diff
@@ -214,18 +214,45 @@ setMethod("toDF", signature(x = "RDD"),
 #' sc <- sparkR.init()
 #' sqlContext <- sparkRSQL.init(sc)
 #' path <- "path/to/file.json"
-#' df <- jsonFile(sqlContext, path)
+#' df <- read.json(sqlContext, path)
 #' }
 
-jsonFile <- function(sqlContext, path) {
+read.json <- function(sqlContext, ...) {
```

Comment:
we can keep parquetFile as is, and mark it as deprecated. And then add read.parquet which will call DataFrame.read.parquet.